config.json

```diff
@@ -1,836 +1,793 @@
-      "slug": "go-counting",
       "core": false,
-      "unlocked_by": null,
       "difficulty": 9,
+      "slug": "go-counting",
       "topics": [
-        "Maybe"
-      ]
+        "maybe"
+      ],
+      "unlocked_by": null,
+      "uuid": "8f1854d7-b8cb-4dfa-9e6a-8811b0891798"
     },
     {
-      "uuid": "056d6e6c-4d07-4bc0-b0d0-fb76a9ec8672",
-      "slug": "lens-person",
       "core": false,
-      "unlocked_by": null,
       "difficulty": 9,
-      "topics": [
-      ]
+      "slug": "lens-person",
+      "topics": null,
+      "unlocked_by": null,
+      "uuid": "056d6e6c-4d07-4bc0-b0d0-fb76a9ec8672"
     },
     {
-      "uuid": "e585c482-2088-40bf-bc97-bc6fac6a7a72",
-      "slug": "zipper",
       "core": false,
-      "unlocked_by": null,
       "difficulty": 10,
+      "slug": "zipper",
       "topics": [
-        "Define type",
-        "Maybe"
-      ]
+        "define_type",
+        "maybe"
+      ],
+      "unlocked_by": null,
+      "uuid": "e585c482-2088-40bf-bc97-bc6fac6a7a72"
     },
     {
-      "uuid": "04c86a05-4d34-4a20-9489-db744c205bb6",
-      "slug": "forth",
       "core": false,
-      "unlocked_by": null,
       "difficulty": 10,
+      "slug": "forth",
       "topics": [
-        "Define type",
-        "Either",
-        "Stack"
-      ]
+        "define_type",
+        "either",
+        "stack"
+      ],
+      "unlocked_by": null,
+      "uuid": "04c86a05-4d34-4a20-9489-db744c205bb6"
     },
     {
-      "uuid": "010582a2-4c44-44fb-ac6b-bf1546d303a3",
-      "slug": "pov",
       "core": false,
-      "unlocked_by": null,
       "difficulty": 10,
+      "slug": "pov",
       "topics": [
-        "Maybe"
-      ]
+        "maybe"
+      ],
+      "unlocked_by": null,
+      "uuid": "010582a2-4c44-44fb-ac6b-bf1546d303a3"
     },
     {
-      "uuid": "48ca1682-ca1b-4a2c-be55-624620f336a7",
+      "deprecated": true,
       "slug": "binary",
-      "deprecated": true
+      "uuid": "48ca1682-ca1b-4a2c-be55-624620f336a7"
     },
     {
-      "uuid": "317ef3e7-67ce-4940-bc90-eed1c4a27c19",
+      "deprecated": true,
       "slug": "gigasecond",
-      "deprecated": true
+      "uuid": "317ef3e7-67ce-4940-bc90-eed1c4a27c19"
     },
     {
-      "uuid": "964c07aa-abca-49b8-8428-1b81ee39066d",
+      "deprecated": true,
       "slug": "hexadecimal",
-      "deprecated": true
+      "uuid": "964c07aa-abca-49b8-8428-1b81ee39066d"
     },
     {
-      "uuid": "2566f8ee-28cf-4429-9ced-42759c810d7c",
+      "deprecated": true,
       "slug": "octal",
-      "deprecated": true
+      "uuid": "2566f8ee-28cf-4429-9ced-42759c810d7c"
     },
     {
-      "uuid": "d5997b60-e54c-4caa-beae-8614f0da3fb3",
+      "deprecated": true,
       "slug": "trinary",
-      "deprecated": true
+      "uuid": "d5997b60-e54c-4caa-beae-8614f0da3fb3"
     }
   ],
-  "foregone": [
-
-  ]
-}
+  "foregone": [],
+  "ignore_pattern": "example",
+  "language": "Haskell",
+  "solution_pattern": "example.*[.]hs"
```

Comment:
should it not be the case that ignore_pattern and solution_pattern are next to each other, instead of having slug interposed in between?